sympy/core/tests/test_relational.py

```diff
@@ -92,7 +92,7 @@ def test_Eq():
 
     # issue 6116
     p = Symbol('p', positive=True)
-    assert Eq(p, 0) is S.false
```

Comment:
This too. The new test doesn't really test the same thing.

Reply:
similar behaviour here as well.

Reply:
In my current master:
```
In [1]: from sympy import Eq, Symbol, S

In [2]: p = Symbol('p', positive=True)

In [3]: Eq(p, 0) is S.false
Out[3]: True
```

Reply:
ohh thanks, I was missing the `positive`.